test/common/http/header_map_impl_fuzz.proto

```diff
@@ -83,7 +83,6 @@ message Action {
     MutateAndMove mutate_and_move = 12;
     Append append = 11;
     google.protobuf.Empty copy = 7;
-    string lookup = 8;
```

Comment:
Removing fields in fuzz tests will cause all old corpus entries to be invalidated unless unknown fields are allowed. This is only supported in latest sha of libprotobuf-mutator
https://github.com/google/libprotobuf-mutator/commits/master

I can make a separate commit, or lmk if you want to add to this

Reply:
Happy to make the change but I'm not sure what change you want me to make. Can you provide more detail?

Reply:
We need a bump to the latest commit in libprotobuf-mutator! That's all.

Reply:
Sure will do.